resource/ui/ts/util/format.ts

```diff
@@ -0,0 +1,46 @@
+// source: util/format.ts
+/// <reference path="../util/convert.ts" />
+// Author: Bram Gruneir (bram+code@cockroachlabs.com)
+
+/**
+ * Utils contains common utilities.
+ */
+module Utils {
+    /**
+     * Formatter contains common code for converting numbers and dates to human
+     * readable formats.
+     */
+    export module Format {
+        /**
+         * DateFromDate formats a Date object into a human readable date
+         * string.
+         */
+        var _datetimeFormatter = d3.time.format("%Y-%m-%d %H:%M:%S");
+        export function DateFromDate(datetime: Date): string {
```

Comment:
This should just be Format.Date.  Everything in this package is going to output a string.